### PR TITLE
Frontend improvements

### PR DESCRIPTION
**Pull request by adrian-larysz-rmtcfm-com**

### CODE REVIEW

**Review by rafal-ksiazek-rmtcfm-com — APPROVED** [review]

File: client/src/App.vue [hunk 1]
```diff
@@ -24,13 +24,15 @@
 import { mapGetters, mapActions } from 'vuex';
 import Cart from '@/components/Cart';
 import ProductList from '@/components/ProductList';
+import ResetDataBtn from './components/ResetDataBtn.vue';
```
Comment (daniel-pupczyk-rmtcfm-com) [inline client/src/App.vue]: here is `./components` and above is `@/components` - use one

File: client/src/components/CartItem.vue [hunk 2]
```diff
@@ -2,7 +2,10 @@
     <v-container>
         <v-row>
             <v-col class="pa-0" cols="12" sm="4">
-                <v-img :src="require(`../assets/products/${item.id}.jpg`)" />
+                <v-img
+                    class="img"
+                    :src="require(`../assets/products/${item.id}.jpg`)"
```
Comment (daniel-pupczyk-rmtcfm-com) [inline client/src/components/CartItem.vue]: alias

File: client/src/components/CartItem.vue [hunk 3]
```diff
@@ -89,3 +92,12 @@ export default {
     }
 };
 </script>
+
```
Comment (daniel-pupczyk-rmtcfm-com) [inline client/src/components/CartItem.vue:95]: if you need good score at google then move it to external file

File: client/src/components/Product.vue [hunk 4]
```diff
@@ -1,14 +1,17 @@
 <template>
     <v-col class="product text-center" cols="12" sm="6" md="4">
         <div class="product-img">
-            <v-img :src="require(`../assets/products/${product.id}.jpg`)" />
+            <v-img
+                class="img"
+                :src="require(`../assets/products/${product.id}.jpg`)"
```
Comment (daniel-pupczyk-rmtcfm-com) [inline client/src/components/Product.vue]: alias

File: client/src/components/Product.vue [hunk 5]
```diff
@@ -28,3 +31,12 @@ export default {
     }
 };
 </script>
+
+<style scoped>
+.img {
```
Comment (daniel-pupczyk-rmtcfm-com) [inline client/src/components/Product.vue]: if you use this twice you can add global class for this

File: client/src/components/ResetDataBtn.vue [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<template>
+    <v-btn v-if="cartItems.length > 0" class="mx-auto" @click="resetData">
```
Comment (daniel-pupczyk-rmtcfm-com) [inline client/src/components/ResetDataBtn.vue]: `v-if="cartItems.length"`

**Pull request merged**